articles/zk-snarks-under-the-hood/zk_snarks_under_the_hood.md

```diff
@@ -0,0 +1,139 @@
+---
+Title: "Zk-SNARKs: Under the Hood"
+Status: Done
+Level: 6
+Note: There is a few places that are hard to fully understand
+---
+
+# Introduction
+
+Remember that the solution to a QAP is a set of polynomials (A, B, C) such that A(x) * B(x) - C(x) = H(x) * Z(x), where:
```

Comment:
Link QAP to a new markdown file. And from that file, create a link to Vitalik's blog

Reply:
![Uploading Screenshot 2024-01-06 at 18.39.31.png…]()

Reply:
Fix the math